core/contextmenu_items.ts

```diff
@@ -631,7 +631,7 @@ export function registerCommentCreate() {
           workspace,
         ),
       );
-      common.setSelected(comment);
+      getFocusManager().focusNode(comment);
```

Comment:
Verify that this works.

Reply:
This seems to be working correctly. How I verified: I added a workspace comment and verified that its border was highlighted (indicating that it was selected). Removing this line also avoids the auto-selection upon creation. Focusing a workspace comment does NOT mean it can be edited for text input.

All of this seems to match existing behavior.